src/Compilers/VisualBasic/Test/Semantic/Diagnostics/OperationAnalyzerTests.vb

```diff
@@ -1216,12 +1220,8 @@ End Class
             comp.VerifyDiagnostics()
             comp.VerifyAnalyzerDiagnostics({New ParamsArrayTestAnalyzer}, Nothing, Nothing, False,
                                            Diagnostic(ParamsArrayTestAnalyzer.LongParamsDescriptor.Id, "M0(1, 2, 3, 4, 5)").WithLocation(9, 9),
```

Comment:
Would it be better to temporarily disable tests, if correct implementation is expected soon?